bigtop-packages/src/rpm/clickhouse-jdbc-bridge/SPECS/clickhouse-jdbc-bridge.spec

```diff
@@ -36,8 +42,10 @@ bash %{SOURCE1} %{clickhouse_jdbc_bridge_version}
 /bin/bash %{SOURCE2} %{buildroot} %{clickhouse_jdbc_bridge_version}
 
 %post
-getent group clickhouse-jdbc-bridge >/dev/null || groupadd -r clickhouse-jdbc-bridge
-getent passwd clickhouse-jdbc-bridge >/dev/null || useradd -c "clickhouse-jdbc-bridge" -d /usr/share/clickhouse-jdbc-bridge -r -M clickhouse-jdbc-bridge 2> /dev/null || :
+CHJBUSER="clickhouse-jdbc-bridge"
```

Comment:
Why did you have to do this?

Reply:
1. Using of variables in repeatable string is a good practice. 
2. It is not possible to add user without group (-g) specifying in altlinux if that group is already exists.